modules/developer-cli-odo-adding-storage-to-the-application-components.adoc

```diff
@@ -1,13 +1,11 @@
 // Module included in the following assemblies:
 //
 // *cli_reference/developer_cli_odo/creating-an-application-with-odo.adoc
+// *cli_reference/developer_cli_odo/working-with-storage.adoc
 
 [id="adding-storage-to-the-application-components_{context}"]
 = Adding storage to the application components
 
-Persistent storage keeps data available between restarts of {odo-title}.
```

Comment:
You need to keep an introduction statement for the module. I think the second and third sentences should be in the module instead of the assembly.

Reply:
Thank you! Added this sentence as an intro to the assembly and kept the rest as an intro to the module itself.